crates/sl-messages/src/message.rs

```diff
@@ -214,6 +206,10 @@ impl<'a> TryFrom<&'a [u8]> for MsgHdr {
 }
 
 impl MsgHdr {
+    pub const MAX_TTL: u64 = (0xffff - 255) * 10 + 250;
```

Comment:
where does that come from?

Reply:
TTL is encoded into 16 bits. The first 255 seconds are encoded as is, then rounded up to the next multiple of 10: 260, 270, ...
MAX_TTL is the maximum TTL that could be encoded using this scheme. Or about 7 days.